From 76d7031d135013c82788ee6d781ad43b517430bb Mon Sep 17 00:00:00 2001
From: Lupia <129526603+Lupiax@users.noreply.github.com>
Date: Sun, 2 Apr 2023 19:00:21 +0300
Subject: [PATCH] Include missing header.

Including the cstdint header will fix the build on latest Fedora 38.
---
 alvr/vulkan_layer/layer/settings.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/alvr/vulkan_layer/layer/settings.h b/alvr/vulkan_layer/layer/settings.h
index 8932a7adb..233fc268b 100644
--- a/alvr/vulkan_layer/layer/settings.h
+++ b/alvr/vulkan_layer/layer/settings.h
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <string>
+#include <cstdint>
 
 class Settings
 {
